website/www/site/layouts/shortcodes/playground.html

```diff
@@ -10,10 +10,13 @@
 limitations under the License. See accompanying LICENSE file.
 */}}
 
-<iframe
-    src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled={{ .Get 0 }}&example={{ .Get 1 }}&code={{ .Get 2 }}"
-    width="100%"
-    height="{{ .Get 3 }}"
-    class="code-snippet playground"
-    allow="clipboard-write">
-</iframe>
+<div id="playgroundIframeContainer">
```

Comment:
`id` should not be used in shortcode because the shortcode can be used multiple times on a page. It actually will be used many times since we need to convert many doc highlighted examples to playground instances.

Reply:
done